### DISCUSSION TITLE
Plans to add icon customization via script to Onboarding?

### DISCUSSION THREAD

**Discussion by mm2270** [IBM/mac-ibm-notifications#55] [Ideas]
Title basically says it all. I'm wondering if there are future plans to add the ability to customize the Onboarding icon when calling IBM Notifier in a script.

I know right now the app code can be updated to include a custom icon and then rebuilt and notarized, etc. The problem for me is, I don't have a company developer account and don't have the ability to easily sign and/or notarize the updated application as a result. I could try using the unsigned app, but I don't believe that's an advisable course to take.

This is somewhat related to [Issue 44](https://github.com/IBM/mac-ibm-notifications/issues/44)

**Comment by SMartorelli** [comment]
Hi, yes we're working on integrate this feature in the next release 2.4.1 as you can see from this issue #43 💪🏻
This release should drop in the next couple of weeks at last 😁
